### ISSUE TITLE
Brave struggles running weather map website ventusky.com 

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#28545]
## Description 

The site http://ventusky.com/ is a meteorological website that focuses on weather prediction and meteorological data visualisation. Brave struggles to fully load/display the map properly under most settings, although it will intermittently work properly in Brave. Note that this does work in Chrome and other Chromium based browsers. 

## Steps to Reproduce

   1. Visit https://www.ventusky.com/
   2. Experiment with settings on the left-hand side
   3. Observe Brave struggle to load/display weather map properly

## Reproduces how often: 

Easily/intermittently

## Brave version (brave://version info)

v1.48.164

## Version/Channel Information:

- Can you reproduce this issue with the current release? Y
- Can you reproduce this issue with the beta channel? Y
- Can you reproduce this issue with the nightly channel? Y

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? N

## Miscellaneous Information:

Original user thread:
https://community.brave.com/t/problems-with-animation-on-the-site/469039/22
cc @ryanbr

**Comment by ryanbr** [comment]
Allowing fingerprinting fixes the overlays on https://www.ventusky.com/  cc: @pilgrim-brave

**Comment by lolafur** [comment]
> Allowing fingerprinting fixes the overlays on https://www.ventusky.com/ cc: @pilgrim-brave

Good day, I have already tried all the options: disable completely ad blocker built into the browser, disable blocking inter-site cookies, fingerprints, cleaning cache, cookies, reinstalling the browser, creating a new profile. I did all of these things and none of them helped. Also turned off hardware acceleration and it too had no result. The only thing left is to completely reinstall Windows 10, but that's the extreme :))

**Comment by devilsdonotcry** [comment]
I have found this thread looking for tips why Ventusky on Brave browser struggles with loading. On Chrome it works fine. I have just subscribed the thread and wait for some good news :)

**Comment by lolafur** [comment]
> have found this thread looking for tips why Ventusky on Brave browser struggles with loading. On Chrome it works fine. I have just subscribed the thread and wait for some good news :)

I'll tell you more: I tried this site in Edge and Vivaldi browsers. There, too, everything works and loads normally, no problems at all. Only Brave has this problem, let's wait for a fix.

**Comment by ryanbr** [comment]
Try testing in the latest Brave Nightly. https://brave.com/download-nightly/   (1.50.44+)

**Comment by lolafur** [comment]
> 

Okay, I'll try it and report the result here.

**Comment by lolafur** [comment]
> Try testing in the latest Brave Nightly. https://brave.com/download-nightly/ (1.50.44+)

I have good news for myself:) Tested the same site as it works in the night version of the browser. I found no problems, everything loaded perfectly and very quickly. Tested on a clean profile, that is, did not change the settings of the browser and did not put extensions. And then put one extension and slightly changed the settings in the browser. It also worked fine.
